automobile/scenario-creation-tutorial.md

```diff
@@ -0,0 +1,78 @@
+## Creation of a real world scenario
```

Comment:
I would keep the same title as the page name, e.g., "Scenario creation tutorial".